app/controllers/tag_implication_controller.rb

```diff
@@ -1,76 +1,96 @@
+# Filename: tag_implication_controller.rb
+# Description: Provides controller for creating, updating, deleting, and indexing on implicit tags
```

Comment:
this is just rewording the class name and adds no value (if not also confusing things on what "indexing" means, where "deleting" part goes, etc)

Reply:
Looking back at this, this is completely redundant and I agree that this makes the whole class more confusing. I have removed this and added to the `.rubocop.yml` to ignore Documentation which checks to see if there's a header comment.